### ISSUE TITLE
On second vertical monitor in maximized view application redraw randomly and switches between different states 

### ISSUE THREAD

**Issue by r3bers** [telegramdesktop/tdesktop#27746]
### Steps to reproduce

1. Run Telegram
2. Put in maximized view on a second monitor with vertical orientation. And Watch.
3. You can do anything in othes application on other monitors. Telegram switches betwen different states randonly and always without any pattern.

### Expected behaviour

I expect Telegram doing  nothig. Just stady behavior without moving anything inside window.

### Actual behaviour

Redrawing conent inside window and changing width/height of elements randomly.
I don't click on anything in a sample video, just moving mouse. I think this happens on window redraw.
I noticed this behavior long time ago. But newer versions didn't fix anything.

https://github.com/telegramdesktop/tdesktop/assets/11983427/c13aca0e-3310-41aa-84e0-9885025e2094

### Operating system

Wondows 11 x64

### Version of Telegram Desktop

4.16.6 x64

### Installation source

Static binary from official website

**Comment by ilya-fedin** [comment]
I can't reproduce. Is it possible you have some kind of modified/stripped Windows?

**Comment by r3bers** [comment]
Windows as usual, nothing modified. When Telegram window not in maximized state just strech right before edge it is normal.

**Comment by r3bers** [comment]
![Screenshot 2024-04-14 052613](https://github.com/telegramdesktop/tdesktop/assets/11983427/b32f4fe6-59e2-41ef-9ed1-e9c2f67cc242)
All problems only here.
This happens almost two or more years. Windows already updateted many times still the same.

**Comment by ilya-fedin** [comment]
Can you reproduce with just one vertical monitor? I can't. And I don't have more monitors to recreate your environment...

**Comment by r3bers** [comment]
Redrawing (changing width/height) happens when something changes in All Chats Folder, even if you select some quite silent folder, still redrawing on any event

https://github.com/telegramdesktop/tdesktop/assets/11983427/4a40296a-d9f7-4533-8956-bb2bdaab2794

**Comment by r3bers** [comment]
I can't reproduce it on one monitor. Telegram on my main monitor stand still. Maybe it depends on windows system adresses pixels and return parameters of telegram window to app depends on place of monitor on all monitors pixel address space.

**Comment by xNombre** [comment]
I can confirm this bug is also affecting me for quite a time. Stock windows 11 here without any magical modifications.
My screens config:
![obraz](https://github.com/telegramdesktop/tdesktop/assets/25608350/10cbdc6c-6536-4039-acb7-ec770066c7ad)
Im using Gigabyte g24f2 and g24f.

**Comment by xNombre** [comment]
@ilya-fedin Hello, I managed to do bisect and recompile Telegram a few times to find the culprit.
I found out that the last working version is v4.8.10.
With v4.8.11 commit introduced in lib_ui (probably https://github.com/desktop-app/lib_ui/commit/4a41346a6428571ec57422ea86e426f252c8ce39) caused default windows status bar to show up randomly within a telegram window:
![obraz](https://github.com/telegramdesktop/tdesktop/assets/25608350/16b809ba-3aec-451a-9a9e-52bcc8f1c938)
but this doesn't cause said issue (yet).
Everything changed with v4.9.0 where updated lib_ui introduced fixes for window resizing - https://github.com/desktop-app/lib_ui/pull/163 . With these changes windows status bar doesn't show up randomly, however, the discussed issue started its life there.
